ewatercycle/config/_validators.py

```diff
@@ -115,36 +124,30 @@ def validate_path(value, allow_none=False):
 
 validate_string = _make_type_validator(str)
 validate_string_or_none = _make_type_validator(str, allow_none=True)
-validate_stringlist = _listify_validator(validate_string,
-                                         docstring='Return a list of strings.')
+validate_stringlist = _listify_validator(
+    validate_string, docstring="Return a list of strings."
+)
 validate_int = _make_type_validator(int)
 validate_int_or_none = _make_type_validator(int, allow_none=True)
 validate_float = _make_type_validator(float)
-validate_floatlist = _listify_validator(validate_float,
-                                        docstring='Return a list of floats.')
+validate_floatlist = _listify_validator(
+    validate_float, docstring="Return a list of floats."
+)
 
 validate_path_or_none = _make_type_validator(validate_path, allow_none=True)
 
-validate_pathlist = _listify_validator(validate_path,
-                                       docstring='Return a list of paths.')
+validate_pathlist = _listify_validator(
+    validate_path, docstring="Return a list of paths."
+)
+
+validate_dict_parameterset = _make_type_validator(dict, allow_none=True)
 
 _validators = {
-    'esmvaltool_config': validate_path_or_none,
-    'grdc_location': validate_path_or_none,
-    'container_engine': validate_string_or_none,
-    'singularity_dir': validate_path_or_none,
-    'output_dir': validate_path_or_none,
-    'ewatercycle_config': validate_path_or_none,
```

Comment:
This is a breaking change. There's a reload system for the CFG object that uses this. Do we want to get rid of that?

Reply:
Oeps, we should write ewatercycle.yaml without `ewatercycle_config` key.